hdk-rust/wasm-test/src/handle_crud.rs

```diff
@@ -6,11 +6,16 @@ use holochain_wasm_utils::{
 };
 use TestEntryType;
 
+use std::{
+    time::Duration
+};
+
 //
 pub(crate) fn handle_update_entry_ok() -> JsonString {
     // Commit v1 entry
     hdk::debug("**** Commit v1 entry").ok();
     let res = hdk::commit_entry(&hdk_test_entry());
+    hdk::sleep(Duration::from_millis(35000)).unwrap();
```

Comment:
whoa!!  Do you really need to wait 35 seconds for this test to work?  All these sleeps will dramatically increase the wall time on CI.  Can we decrease this?

Reply:
Yeah, we need too find a different solution here, I think...